pkg/polaris/azure.go

```diff
@@ -321,7 +321,7 @@ func (c *Client) AzureSubscriptionSetRegions(ctx context.Context, id AzureSubscr
 		Name: subscription.Name,
 	}}
 
-	status, err := c.gql.AzureCloudAccountUpdate(ctx, graphql.CloudNativeProtection, regionAdd, regionRemove, sub)
+	status, err := c.gql.AzureUpdateCloudAccount(ctx, graphql.CloudNativeProtection, regionAdd, regionRemove, sub)
 	if len(status) != 1 {
```

Comment:
Should `err != nil` be checked ahead of the status length?

Reply:
Agree, should be fixed now.